qutip/control/cy_grape.pyx

```diff
@@ -1,36 +1,3 @@
-#cython: language_level=3
```

Comment:
We should probably keep these `cython: language_level` lines? Or at least they are not part of the license boilerplate.

Reply:
I've updated the PR to skip these removals.  Not sure why these lines haven't gained the "outdated" tag after the push (at least for me), because they should be entirely gone.